### PR TITLE
Add SeoBundle twig extension documentation

### PR DESCRIPTION
**Pull request by benglass**
| Q             | A
| ------------- | ---
| Doc fix?      | no
| New docs?     | yes (https://github.com/symfony-cmf/SeoBundle/pull/263)
| Applies to    | SeoBundle master/1.2

### CODE REVIEW

File: bundles/seo/twig.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Setting Seo Metadata from Twig
+==============================
+
+.. versionadded:: 1.2
+    The twig extension was added in SeoBundle 1.2.
+
+This bundle provides a twig function ``cmf_seo_update_seo_page``
+which lets you set the object which should be used to populate
+the seo page. For details on how this works, read :doc:`introduction`.
```
Comment (dbu) [inline bundles/seo/twig.rst]: s/this/populating the SEO page/

Comment (benglass) [inline bundles/seo/twig.rst]: Fixed

File: bundles/seo/twig.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Setting Seo Metadata from Twig
+==============================
+
+.. versionadded:: 1.2
+    The twig extension was added in SeoBundle 1.2.
+
+This bundle provides a twig function ``cmf_seo_update_metadata``
+which lets you set the object which should be used to populate
```
Comment (dbu) [inline bundles/seo/twig.rst]: is it changing the object or does it just extract information from that object, in addition to what it already extracted from previous objects?

File: bundles/seo/twig.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <body>
+            <p>Some page body.</p>
+        </body>
+    </html>
+
+    <!-- app/Resources/views/Blog/post.html.twig -->
+    {% extends 'base.html.twig' %}
+
+    {% block metadata %}
+        {% do cmf_seo_update_metadata(post) %}
```
Comment (dbu) [inline bundles/seo/twig.rst]: i think the `do` here is too much.

**Comment by dbu** [comment]
please also add a note in the documentation about extracting the metadata that mentions the twig functions in case documents are looped over in twig or otherwise not yet available in the controller.

File: bundles/seo/twig.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
+Setting Seo Metadata from Twig
+==============================
+
+.. versionadded:: 1.2
+    The twig extension was added in SeoBundle 1.2.
+
+This bundle provides a twig function ``cmf_seo_update_metadata``
+which lets you populate the seo page from an object.
```
Comment (benglass) [inline bundles/seo/twig.rst]: @dbu I reworded this per your comment.

File: bundles/seo/introduction.rst [hunk 5]
```diff
@@ -137,6 +137,16 @@ key in ``cmf_seo.content_key``.
 You may also want to disable this listener when implementing your own mechanism
 to extract SEO information.
 
+The Twig Extension
```
Comment (benglass) [inline bundles/seo/introduction.rst]: @dbu I added this here per your comment on mentioning the twig extension

File: bundles/seo/twig.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        <body>
+            <p>Some page body.</p>
+        </body>
+    </html>
+
+    <!-- app/Resources/views/Blog/post.html.twig -->
+    {% extends 'base.html.twig' %}
+
+    {% block metadata %}
+        {{ cmf_seo_update_metadata(post) }}
```
Comment (benglass) [inline bundles/seo/twig.rst]: @dbu Removed the do, you caught me trying to be clever.

**Comment by dbu** [comment]
thanks!

**Pull request merged**